pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -31,17 +37,29 @@ def setUpClass(cls):
         if cls.cfg.pulp_version < Version('2.17'):
             raise unittest.SkipTest('This test requires Pulp 2.17 or newer.')
         cls.client = api.Client(cls.cfg, api.json_handler)
+        cls.repos = []
+        try:
+            for _ in range(2):
+                body = gen_repo()
+                if _ == 0:  # adding feed only to first repo
+                    body['importer_config']['feed'] = RPM_WITH_MODULES_FEED_URL
+                body['distributors'] = [gen_distributor()]
+                repo = cls.client.post(REPOSITORY_PATH, body)
+                cls.repos.append(repo)
+        except:  # noqa:E722
```

Comment:
I would suggest to avoid the use of bare exceptions. Try to create the necessary resources inside a test case, try to use the `self.addCleanup` .